Documentation & Releases.

AJFire14 opened the pull request:
- Releases Page added with release notes.
- Documentation page completed: Snippets, Commands, Observers, Engines & Permissions.
- "Switch Versions" button added to menu.
- Styleguide updated.
- Theme updated.
- URL change.
- Meta tags.
- Minor issues fixed.

- Completed requested changes.
- Sorted home page buttons.

Commit:
Minor Grammatical Changes.

- Added . in intro page doc.

- Fixed spelling of Engine description.

- Added a paragraph on the use of an Engine. (Still in works)

Commit:
Merge branch 'development' of https://github.com/Sparkbots/Spark-documentation into development

Commit:
Receive spelling

Corrected the spelling of "recieve" to "receive".

TobiasFeld22 committed:
Use the new stats api

To fix runkit issues.

Sanil14 committed:
Added Commands page

This adds onto #7 for V0.1

Commit:
Commands - Explanation

- Explanation complete for commands.
- Commands page for documentation now 100% complete.

Commit:
Added Releases

+ Added page releases
+ Added header, menu, tags and links
+ Added structure of Releases page.
+ Tags: Beta, Latest & Outdated(D..).

To be Done (Tobias):
> Links from menu to the titles.
> Release Notes V0.1 & V0.0.6
> Align tags with titles.
> Final adjustments.

Commit:
Cmds & Styleguide

Added boxes to styleguide & disable method for commands.

Commit:
Engines

Completed engines page for docs.

TobiasFeld22 committed:
Updated themes & Removed some useless breaks

Testing out if the all dark/yellow themes are something worth staying on.

Commit:
Err Ignore

Commit:
Observers - doc page

Finished observers doc page + few minor text edits.

TobiasFeld22 committed:
Added release notes for 0.1 & 0.0.6

TobiasFeld22 committed:
Fixed some travis issues

Commit:
Permissions & Snippets

- Completed docs for permissions and snippets.
- Updated styleguide.

TobiasFeld22 committed:
Added switch version button

TobiasFeld22 committed:
Added meta tags

fixes #17

TobiasFeld22 committed:
Changed urls

TobiasFeld22 commented on 404.html:
```diff
@@ -15,8 +15,9 @@ <h1 class="title">
-              <a class="button is-danger">Documentation</a>
+              <a class="button is-danger icon-button"> Homepage </a>
+              <a class="button is-danger icon-button"> Documentation </a>
+              <a class="button is-danger"> Releases </a>
```
Releases shouldn't be shown on a 404 page.

AJFire14 replied:
Complete.

TobiasFeld22 commented on documentation/intro.html:
```diff
@@ -1,17 +1,17 @@
+color: is-dark
 title: Getting Started
-subtitle: Getting started with the Spark framework
+subtitle: Getting started with the Spark framework.
```
Titles & subtitles do not use dots.

AJFire14 replied:
Complete.

TobiasFeld22 commented on documentation/observers.html:
```diff
@@ -1,11 +1,87 @@
-color: is-primary
+color: is-dark
 title: Observers
 subtitle: Observe behaviour of messages or commands.
```
Titles & Subtitles don't use dots

AJFire14 replied:
Complete.

TobiasFeld22 commented on documentation/permissions.html:
```diff
@@ -1,11 +1,104 @@
-color: is-danger
+color: is-dark
 title: Permissions
 subtitle: Lock certain commands to certain people.
```
Titles & Subtitles don't use dots.

AJFire14 replied:
Complete.

TobiasFeld22 commented on documentation/snippets.html:
```diff
@@ -1,11 +1,79 @@
-color: is-warning
+color: is-dark
 title: Snippets
 subtitle: Functions you can use in the whole framework.
```
Titles & Subtitles don't use dots.

AJFire14 replied:
Complete.

TobiasFeld22 commented on documentation/observers.html:
```diff
@@ -1,11 +1,87 @@
+        
+        <ul>
+          <li>
+              <p class="subtitle"> setType(<span class="argument"> Type </span>) </p>
```
To keep things uniform, i suggest keeping all subtitles to `<h4>` as the styleguide suggests.

AJFire14 replied:
Complete.

TobiasFeld22 commented on _layouts/default.html:
```diff
@@ -57,7 +66,7 @@
         <div class="container">
             <div class="content has-text-centered">
-                    <a href="https://github.com/sparkbots/">Sparkbots</a> &copy; 2018 | <a href="https://github.com/TobiasFeld22/Spark">Framework</a> | <a href="https://github.com/Sparkbots/Spark-documentation/">Website</a>
+                <a href="https://github.com/sparkbots/">Sparkbots</a> &copy; 2018 | <a href="https://github.com/TobiasFeld22/Spark">Framework</a> | <a href="https://github.com/Sparkbots/Spark-documentation/">Website</a>
```
Update to `Sparkbots/Spark`

AJFire14 replied:
Complete.

TobiasFeld22 commented on documentation/intro.html:
```diff
@@ -25,15 +25,15 @@ <h1 class="title">Setting up the bot</h1>
 Download a recommended code editor from below:<br><br>
```
Wrap this in `<p>`

AJFire14 replied:
Complete.

TobiasFeld22 commented on documentation/intro.html:
```diff
@@ -94,15 +94,15 @@ <h1 class="title">Starting the bot</h1>
 
 <code>bot-name</code> is the name you previously set in the application. <br> <br>
 
 You have now successfully set up your bot!<br><br>
```
wrap up in `<p>`

AJFire14 replied:
Complete.

TobiasFeld22 reviewed (changes requested):
I like the work that's been done so far. 
Things that i would like to bring up is that subtitles don't use dots, that the getting started page has a lot of lines without `<p>` and with a lot of `<br>` tags and that some links should be updated to reflect repository changes.

Commit:
Final Changes

- Made requested changes (PR)
- Final review of site/minor changes.
- Sorted home page buttons.
